openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/insights/DataInsightsApp.java

```diff
@@ -238,7 +238,7 @@ private WorkflowStats processWebAnalytics() {
     try {
       workflow.process();
     } catch (SearchIndexException ex) {
-      jobData.setStatus(EventPublisherJob.Status.FAILED);
+      jobData.setStatus(EventPublisherJob.Status.RUNNING);
```

Comment:
does the job still continue after this ? @ulixius9

Reply:
Yes